Amplitude/AMPDeviceInfo.m

```diff
@@ -180,6 +207,75 @@ + (NSString*)getVendorID:(int) maxAttempts
     } else {
         return identifier;
     }
+#else
```

Comment:
Will this code ever run on OS X? Looks like the call to getVendorID:maxAttempts in getVendorId is compiled out.